CareKit/CareKit/iOS/Task/ChecklistTaskView.swift

```diff
@@ -0,0 +1,216 @@
+    }
+
+    /// Handle any errors that may occur.
+    /// - Parameter handler: Handle the encountered error.
+    public func onError(_ perform: @escaping (Error) -> Void) -> Self {
+        .init(taskView: .init(copying: taskView, settingErrorHandler: perform))
+    }
+}
+
+@available(iOS 14.0, watchOS 7.0, *)
+public extension ChecklistTaskView where Header == _ChecklistTaskViewHeader, Content == _ChecklistTaskViewContent<ChecklistItem, Text> {
+
+    /// Create an instance that displays the default content. The first task and event that match the provided queries will be fetched from the the
+    /// store and displayed in the view. The view will update when changes occur in the store.
+    /// - Parameters:
+    ///     - taskID: The ID of the task to fetch.
+    ///     - eventQuery: A query used to fetch an event in the store.
+    ///     - storeManager: Wraps the store that contains the task and event to fetch.
+    ///     - content: Create a view to display whenever the body is computed.
+    init(taskID: String, eventQuery: OCKEventQuery, storeManager: OCKSynchronizedStoreManager) {
+        self.init(taskID: taskID, eventQuery: eventQuery, storeManager: storeManager) {
+            .init(viewModel: $0.viewModel)
+        }
+    }
+
+    /// Create an instance that displays the default content. The first event that matches the provided query will be fetched from the the store and
+    /// displayed in the view. The view will update when changes occur in the store.
+    /// - Parameters:
+    ///     - task: The task associated with the event to fetch.
+    ///     - eventQuery: A query used to fetch an event in the store.
+    ///     - storeManager: Wraps the store that contains the event to fetch.
+    ///     - content: Create a view to display whenever the body is computed.
+    init(task: OCKAnyTask, eventQuery: OCKEventQuery, storeManager: OCKSynchronizedStoreManager) {
+        self.init(task: task, eventQuery: eventQuery, storeManager: storeManager) {
+            .init(viewModel: $0.viewModel)
+        }
+    }
+
+    /// Create an instance that displays the default content.
+    /// - Parameters:
+    ///     - controller: Controller that holds a reference to data displayed by the view.
+    init(controller: OCKChecklistTaskController) {
+        taskView = .init(controller: controller) {
+            .init(viewModel: $0.viewModel)
+        }
+    }
+}
+
+private extension CareKitUI.ChecklistTaskView where Header == _ChecklistTaskViewHeader, Content == _ChecklistTaskViewContent<ChecklistItem, Text> {
+
+    init(viewModel: ChecklistTaskViewModel?) {
+        self.init(
+            title: Text(viewModel?.title ?? ""),
+            detail: viewModel?.detail.map(Text.init),
+            items: viewModel?.items ?? [],
+            action: viewModel?.action ?? { _ in },
+            instructions: viewModel?.instructions.map(Text.init)
+        ) { item in
+            Text(item.title)
```

Comment:
This initializer should delegate to an initializer on the view in CareKitUI that builds the default view using the view model. It's nice to have an initializer like that, in case the developer just wants the view as it comes out of the box. I think as things are set up right now, there is no initializer for the default view - the initializer for the default view will not provide any View Builders parameters.

If you decide to go with the architecture I mention below, the initializer will look like:

```swift
init(
    title: Text,
    detail: Text? = nil,
    instructions: Text? = nil,
    items: [ChecklistTaskViewItem],
) { 
    ....
}
```